src/js/base/core/agent.js

```diff
@@ -41,18 +41,18 @@ define(['jquery'], function ($) {
   var isEdge = /Edge\/\d+/.test(userAgent);
 
   var hasCodeMirror = !!window.CodeMirror;
-  if (!hasCodeMirror && isSupportAmd && require) {
-    if (require.hasOwnProperty('resolve')) {
+  if (!hasCodeMirror && isSupportAmd && typeof require !== 'undefined') {
+    if (typeof require.resolve !== undefined) {
```

Comment:
This line should be changed with `type of require.resolve !== 'undefined'` like a 54 line.